devDocs/startupShutdown.md

```diff
@@ -4,14 +4,23 @@
 
 1. For an installed copy:
     1. Ctrl+Alt+N (Desktop shortcut)
+        - can be tested by:
+            1. Set the NVDA config to ensure the welcome dialog loads on startup
+            1. Have an existing NVDA process running with a known handle
+            1. Press (or emulate) Ctrl+Alt+N
+            1. Ensure the known process ends
+            1. Wait until the welcome dialog loads

```

Comment:
Please add the detailed test plan to a new section. It's useful to to have an over-view for quickly mapping out the user processes, and a details for exactly how to test. This test procedure seems very programmatic, instead imagine writing instructions for a non-programmer human. Likely very much like the steps-to-reproduce you'll fins on an issue.

I'd put it before `## Technical notes`:

```
## Manual testing
Instructions for testing startup / shutdown.

### Start from shortcut
Prerequisites:
 - NVDA installed
 - Shortcut enabled during installation

Steps:
 1. Press (or emulate) Ctrl+Alt+N, observe NVDA starts up

Variation:
- At step 1. A version of NVDA is already running. Observe running version exists before installed version starts up.

### Windows Sign-in screen, automatic up
  

## Technical notes
```